_includes/main.liquid

```diff
@@ -60,6 +61,14 @@
     </head>
 
     <body class="body">
+        <div id="notification" key="dev-survey-2021">
+            <div class="notification-info">
+                <span id="notification-close">X</span>
+                <span class="notification-text">
+                    Help Chainlink by taking a <b><a href="https://chainlinkcommunity.typeform.com/to/V2z25k8N">1 minute survey</b></a>!
```

Comment:
I think the link should be this one: https://chn.lk/dev-survey-2021

Reply:
> @dmitrydao What do you think?

Love it overall. It doesn't appear on the page reload though. I assume this is the expected behavior, right?

Reply:
Sounds good. I'm get that fixed up. How's the functionality?

Reply:
I think it's great, especially the bouching effect.

Reply:
It doesn't appear or doesn't bounce? Expected behavior is

1. First load, it slides in to draw attention
2. Future loads, simply present
3. X it out, never see it again

Reply:
It doesn't appear when you close it but that is the expected behavior so I think we're good.